gslib/tests/test_cp.py

```diff
@@ -1002,6 +1002,44 @@ def _Check2():
 
     _Check2()
 
+  @SkipForXML('No compressed transport encoding support for the XML API.')
```

Comment:
Actually, I have one more questions before I merge this -- it looks like the skip explanation message here is for tests that use transport encoding... why are we actually skipping this for the XML API?   Did you intend to skip this for the XML API and just forget to change the message, or did you copy/paste this and not forget to remove the skip decorator entirely?

Reply:
FYI, if I remove this decorator, the test fails when using the XML API.  Looks like you might just need to tweak the regex.  The metadata shows up like this when doing an `ls -L` using the JSON API:

```
    Metadata:               
        X-Goog-Request-Reason: some-reason-here
```

but shows up like this when using the XML API:

```
    Metadata:               
        x-goog-request-reason:some-reason-here
```

Reply:
Thank you for noticing that. I add that decorator, because as you know, the test fails when using the XML API. 
Actually I don't find the output of metadata when using the XML API